api/models/release.go

```diff
@@ -389,8 +390,8 @@ func (r *Release) Formation() (string, error) {
 	return app.Formation(*manifest)
 }
 
-func (r *Release) resolveLinks(app App, manifest *manifest.Manifest) (*manifest.Manifest, error) {
-	m := *manifest
+func (r *Release) resolveLinks(app App, m *manifest.Manifest) (*manifest.Manifest, error) {
+	mp := *m
```

Comment:
Ran into one more existing variable name conflict here - renamed the pointer to the manifest `mp` and updated other references.